src/nn/Model.js

```diff
@@ -58,6 +58,13 @@ export default class Model {
     return ResultCode.NO_ERROR;
   }
 
+  /**
+   * Determine if the OperandCode of model is quant8.
+   */
+  isQuant8() {
+    return this._operands[this._inputs[0]].type === OperandCode.TENSOR_QUANT8_ASYMM ? true : false;
```

Comment:
Probably checking `inputs[0]` is not enough, since later we'll support `QUANTIZE` op that allows fp32 input. How about we check every operand when invoking `addOperand`. We can use a boolean property to record whether there are nay quant8 operand. Please also check all quantization types that include `TENSOR_QUANT8_SYMM_PER_CHANNEL` and `TENSOR_QUANT8_ASYMM_SIGNED`.

Reply:
If only check the operation `add` seems will cause incompatibility in test, because if test other operation there is no `add` in model. So I'll keep this for single operation test and follow your suggestion for example. And `TENSOR_QUANT8_SYMM_PER_CHANNEL` deesn't exist when I create this PR, so I'll create a new PR to add this when tfjs-wasm beckend is ready to converge.

Reply:
> And `TENSOR_QUANT8_SYMM_PER_CHANNEL` deesn't exist when I create this PR

You can rebase your PR to latest master branch and you'll get this new operand type.